redisdb/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG - redisdb
 
+1.1.1 / 2017-08-11
+==================
+
+### Changes
+
+* [IMPROVEMENT] adds "redis.net.commands.instantaneous" metric [#672][].
```

Comment:
Please add the link to you PR #672 at the bottom of the file (see other CHANGELOG for example). And you could use [pimpmychangelog](https://github.com/pcreux/pimpmychangelog) ruby tool if you want.